### DISCUSSION TITLE
It is possible select multiple areas in one image ?

### DISCUSSION THREAD

**Discussion by mcacrz** [advanced-cropper/vue-advanced-cropper#162] [Q&A] [answered]
Hi.
I would to know if is possible select multiples areas in a same image to get his coordinates.

Thanks.

**Comment by Norserium** [comment, marked as answer]
@mcacrz, you can create multiple croppers to choice multiple areas or choice multiple areas sequentially one after another. But you can't simultaneously choice the multiple area in the same cropper. 

If it were possible it would be the completely different cropper. Moreover it produces too much questions, that don't have answers. What if the areas intersect? How to identify the areas? How to specify the count of areas? How to specify the default positions of areas?

So I don't see that it will be possible in this library ever. But, anyway, what's result do you want to get? Perhaps, I can provide some ideas how to implement this (regardless of this library).

**Reply by mcacrz** [reply]
thanks for help

**Reply by Norserium** [reply]
@mcacrz, actually, I didn't help yet. In [the previous similar question](https://github.com/Norserium/vue-advanced-cropper/issues/50#issuecomment-675413923), its author wrote that he used [Vue Movable](https://github.com/daybrush/moveable/tree/master/packages/vue-moveable) to accomplish it. Perhaps, it may be useful for you.
